notebooks/003-hello-segmentation/003-hello-segmentation.ipynb

```diff
@@ -69,18 +69,20 @@
    "execution_count": null,
    "source": [
     "# The segmentation network expects images in BGR format\n",
-    "image = cv2.cvtColor(cv2.imread(\"data/empty_road_mapillary.jpg\"), cv2.COLOR_RGB2BGR)\n",
+    "image = cv2.imread(\"data/empty_road_mapillary.jpg\")\n",
     "\n",
     "# N,C,H,W = batch size, number of channels, height, width\n",
     "N, C, H, W = net.input_info[input_layer_ir].tensor_desc.dims\n",
     "\n",
     "# OpenCV resize expects the destination size as (width, height)\n",
-    "resized_image = cv2.resize(image, (W, H))  \n",
+    "resized_image = cv2.resize(image, (W, H))\n",
+    "RGB_resized_image = cv2.cvtColor(resized_image, cv2.COLOR_BGR2RGB)\n",
+    "\n",
     "# reshape to network input shape\n",
     "input_image = np.expand_dims(\n",
     "    resized_image.transpose(2, 0, 1), 0\n",
     ")  \n",
-    "plt.imshow(image)"
+    "plt.imshow(RGB_resized_image)"
```

Comment:
This works well here, because the network aspect ratio is similar to the image aspect ratio. But that is not always the case - you may have images of 1024x768 and a network input shape of 512x512. Showing the resized image looks strange then. So for the general case, it is often useful to show the image in the original size and resize the mask to the original image size.

Reply:
Created rgb_image instead of RGB_resized_image, now it is rgb copy of base image. Other operation I added is resizing mask to meet H and W for base image.